### ISSUE TITLE
during training of NeMo ASR , model not able to save checkpoint at the end of every epoch

### ISSUE THREAD

**Issue by shahbaznazami** [NVIDIA/NeMo#1944]
**Describe the bug**

when i try to train the NeMo ASR model with the help of pre-train model quartznet(QuartzNet15x5Base-En) . model not saving checkpoint at the end of every epoch. 

**Steps/Code to reproduce bug**

Please list *minimal* steps or code snippet for us to be able to reproduce the bug.

A  helpful guide on on how to craft a minimal bug report  http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports. 

**Expected behavior**

model save chechpoint at end of every epoch.

**Environment details**

If NVIDIA docker image is used you don't need to specify these.
Otherwise, please provide:
- Ubuntu 18.04
- PyTorch 1.1.4
- Python 3.6

**Additional context**

Example: GPU model

**Comment by nithinraok** [comment]
To save checkpoints at every epoch  pass `+exp_manager.checkpoint_callback_params.save_top_k=num_epochs` while calling script or add through yaml file

**Comment by VahidooX** [comment]
nemo saves the last checkpoint, not the ones at the end of each epoch. It can also saves the top K best ones. You need to set K along with what metric to monitor and if best value is min or max. You may add it to your config file:

exp_manager:
  checkpoint_callback_params:
    # in case of multiple validation sets, first one is used
    monitor: "val_wer"
    mode: "min"
    save_top_k: 3
